server/auth_function/test/conftest.py

```diff
@@ -80,8 +78,8 @@ def db_pg_transaction(db_pg_connection, monkeypatch):
 
 # Load a sample cognito test event if necessary
 @pytest.fixture(scope="session")
-def cognito_event():
-    event_file_path = os.path.join(os.path.dirname(__file__), "login_event.json")
+def cognito_event(request):
```

Comment:
Just curious, I assume "request" isn't the http request but the pytest internal object?

Reply:
this is how we can read the parameter passed to fixture method